README.md

```
@@ -53,13 +55,62 @@ A proxy server that enables Cursor IDE to use GitHub Copilot's API instead of Cu
 ## 🔌 Configuration with Cursor IDE
 
 1. Open Cursor IDE
-2. Go to Settings > API Keys
-3. In the "Override OpenAI Base URL" section, enter:
+2. Open the OpenAI-compatible provider settings for your custom model endpoint
+3. Set the base URL to:
    ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)